apps/microsoft-teams/frontend/package.json

```diff
@@ -8,6 +8,7 @@
     "@contentful/f36-icons": "^4.27.0",
     "@contentful/f36-tokens": "4.0.4",
     "@contentful/integration-component-library": "^1.1.41",
+    "@contentful/integration-frontend-toolkit": "file:~/contentful-integration-frontend-toolkit-1.0.0-semantic-release.tgz",
```

Comment:
This obviously needs to be updated with the latest version of the frontend-toolkit install! (along with the associated package.lock changes)